projects/assets/src/translations/translation-chunks-config.ts

```diff
@@ -1,4 +1,4 @@
-interface TranslationChunksConfig {
+export interface TranslationChunksConfig {
```

Comment:
It's not in the scope of this PR.
```suggestion
interface TranslationChunksConfig {
```